Pipfile.lock

```diff
@@ -1,14 +1,13 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "d7d74256d2dbd665591ca1ce111f3d88daf9959a2ac3ba2cca915eaa6cfe359a"
+            "sha256": "00b26fac666fafa35b14a8577c9ba370e462e2fdf32be447288d762fe2bfe21a"
         },
         "pipfile-spec": 6,
         "requires": {
             "python_version": "3.11"
         },
         "sources": [
-            {
```

Comment:
I think you've accidentally deleted this line, which might be what's causing the tests to fail 🙂